include/dpp/json_interface.h

```diff
@@ -44,7 +44,7 @@ namespace dpp {
 		 * @param j nlohmann::json object
 		 * @return T& Reference to self for fluent calling
 		 */
-		T& fill_from_json(nlohmann::json* j) {
+		T& fill_from_json([[maybe_unused]] nlohmann::json* j) {
```

Comment:
how is this ever unused? we always pass json in.

Reply:
It's unused in this definition, causing a warning